### ISSUE TITLE
Enhancement: Copy location of selected (point) symbols to clipboard

### ISSUE THREAD

**Issue by sfroyen** [OpenOrienteering/mapper#1147]
This was discussed a couple of years ago  and I implemented some code to make it happen. Here's a patch set for the current version of Mapper. It's a simple change that I find to be useful.

If the code is accepted, the display coordinate code duplicates code elsewhere and should be factored out.

Mapper Version: 0.9
Operating System: Any

[patch-src_gui_map_map_editor_cpp.txt](https://github.com/OpenOrienteering/mapper/files/2447618/patch-src_gui_map_map_editor_cpp.txt)
[patch-src_gui_map_map_editor_h.txt](https://github.com/OpenOrienteering/mapper/files/2447620/patch-src_gui_map_map_editor_h.txt)

**Comment by dg0yt** [comment]
Are you awary that you can copy the mouse position with right click?

**Comment by sfroyen** [comment]
Yes, but only for a singe point. My change does multiple points. Here a sample output of some GPS points that I collected using the OOM tablet version. Originally, I was using this to collect vegetation sample points to use with Terje Mathisen’s Lidar tools for automated vegetation maps.

39.786354 -105.059008 890 GPS point
39.786440 -105.058898 890 GPS point
39.786532 -105.058931 890 GPS point
39.785704 -105.059018 890 GPS point
39.785706 -105.059149 890 GPS point
39.786325 -105.056111 890 GPS point
39.786166 -105.054879 890 GPS point
39.786157 -105.054043 890 GPS point
39.786222 -105.053968 890 GPS point
39.786271 -105.053952 890 GPS point
39.786073 -105.054191 890 GPS point
39.786143 -105.054134 890 GPS point
39.786170 -105.054953 890 GPS point
39.786350 -105.053939 890 GPS point
39.786494 -105.054030 890 GPS point
39.786566 -105.054189 890 GPS point
39.786529 -105.054199 890 GPS point
….

It’s very tedious to do it one point at a time.

Regards,
Sverre
